src/jit/gcencode.cpp

```diff
@@ -3844,13 +3844,18 @@ struct InterruptibleRangeReporter
     }
 };
 
-void GCInfo::gcMakeRegPtrTable(GcInfoEncoder* gcInfoEncoder,
-                               unsigned       codeSize,
-                               unsigned       prologSize,
-                               MakeRegPtrMode mode)
+void GCInfo::gcMakeRegPtrTable(
+    GcInfoEncoder* gcInfoEncoder, unsigned codeSize, unsigned prologSize, MakeRegPtrMode mode, unsigned* callCntRef)
 {
     GCENCODER_WITH_LOGGING(gcInfoEncoderWithLog, gcInfoEncoder);
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+    const bool fastPath = (compiler->opts.MinOpts() && !compiler->opts.jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT) &&
```

Comment:
>fastPath [](start = 15, length = 8)

`fastPath` seems too generic a name. Why not `trackGcRefs` or `MinOptsTrackGcRefs`?